meson.build

```diff
@@ -25,6 +26,13 @@ if get_option('enable-tizen')
     add_project_arguments('-D__FEATURE_CHECK_SUPPORT__', language: ['c', 'cpp'])
   endif
 endif
+if get_option('enable-yocto')
+  if build_platform == ''
+    build_platform = 'yocto'
+  else
+    error(multi_build_platform_err_msg_template.format(build_platform, 'yocto'))
```

Comment:
This is little bit strange. is there no case in which the build_paltform='yocto' ?? If it is, then there will be error.

Reply:
Logic of 29~35 is overly complex and is not general (extensible). Simplify it.

```
count = 0
if TIZEN
   ...
   count ++
endif
if YOCTO
   ...
   count ++;
endif

if (count > 1) error
```
should be enough.